cpu/msp430_common/Makefile.include

```diff
@@ -1,13 +1,9 @@
-PSEUDOMODULES += msp430_malloc
-
 INCLUDES += -I$(RIOTCPU)/msp430_common/include/
 
 CFLAGS += -DCPU_MODEL_$(call uppercase_and_underscore,$(CPU_MODEL))
 
-export UNDEF += $(BINDIR)/msp430_common/startup.o
-export USEMODULE += msp430_common msp430_common_periph msp430_malloc
-
-DEFAULT_MODULE += oneway_malloc
+UNDEF += $(BINDIR)/msp430_common/startup.o
+USEMODULE += msp430_common msp430_common_periph periph_common
```

Comment:
`newlib_nano` is also required.

Reply:
I think the TI toolchain defaults to newlib_nano. It compiles to the same binary sizes as my self compiled gcc-9.2, which uses `--enable-newlib-nano-formatted-io --enable-newlib-nano-malloc` as newlib configure flags.

Reply:
Yes, but you have to tell the build system that `newlib_nano` is used, otherwise `tests/libc_newlib` fails. Using
```
USEMODULE=newlib_nano make BOARD=wsn430-v1_3b tests/libc_newlib
```
works perfect.

Reply:
@cladmi's explaination in https://github.com/RIOT-OS/RIOT/pull/12133#issuecomment-527094777

Reply:
Argl, I tried "newlib-nano", it couldn't be found and instead of realizing that there's a typo I started looking for newlib-nano.a in the toolchain... Fixed.